### ISSUE TITLE
Method VerumConsilium\Browsershot\Screenshot::fit() does not exists

### ISSUE THREAD

**Issue by Binternet** [verumconsilium/laravel-browsershot#9]
It looks like the `fit` method is not chained:

```
    return \VerumConsilium\Browsershot\Facades\Screenshot::loadUrl('https://example.com')
    ->windowSize(1920, 1080)
    ->fit('contain', 200, 200)
    ->download();
```

This returns:
```
BadMethodCallException
Method VerumConsilium\Browsershot\Screenshot::fit() does not exists
```

**Comment by sgtpepper9907** [comment]
It looks like Browsershot does not provide a `fit` method. Can you tell me where did you get this method from?

**Comment by Binternet** [comment]
https://github.com/spatie/browsershot
Search for 'fit' and you'll see it in the examples

**Comment by sgtpepper9907** [comment]
The `fit()` method (allong with some others) are part of the [Spatie\Image](https://github.com/spatie/image) package, now the other manipulations are available with the new release #10
